package.nls.json

```diff
@@ -0,0 +1,96 @@
+    "cmake-tools.configuration.cmake.debugConfig.externalConsole.description": "Launch an external console for the program",
+    "cmake-tools.configuration.cmake.debugConfig.logging.description": "Tell what types of messages should be logged to the console",
+    "cmake-tools.configuration.cmake.debugConfig.visualizerFile.description": ".natvis file to be used when debugging",
+    "cmake-tools.configuration.cmake.debugConfig.args.description": "Arguments to pass to program command line",
+    "cmake-tools.configuration.cmake.debugConfig.cwd.description": "Set the working directory for the program",
+    "cmake-tools.configuration.cmake.debugConfig.environment.name.description": "Name of environment variable",
+    "cmake-tools.configuration.cmake.debugConfig.environment.value.description": "Value for the environment variable",
+    "cmake-tools.configuration.cmake.debugConfig.miDebuggerPath.description": "Path to gdb or lldb debugger",
+    "cmake-tools.configuration.cmake.debugConfig.stopAtEntry.description": "Stop at the entry point of the target",
+    "cmake-tools.configuration.cmake.debugConfig.setupCommands.description": "Command to set up gdb or lldb",
+    "cmake-tools.configuration.cmake.debugConfig.setupCommands.text.description": "Command to run",
+    "cmake-tools.configuration.cmake.debugConfig.setupCommands.description.description": "Description of the command",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.description": "The build type",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.debug.long": "Emit debug information without performing optimizations",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.reelase.long": "Enable optimizations, omit debug info",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.minsize.long": "Optimize for smallest binary size",
+    "cmake-tools.configuration.cmake.defaultVariants.buildType.reldeb.long": "Perform optimizations AND include debugging information",
+    "cmake-tools.configuration.cmake.ctestArgs.description": "Arguments to pass to CTest",
+    "cmake-tools.configuration.cmake.environment.description": "Environment variables to set when running CMake commands",
+    "cmake-tools.configuration.cmake.environment.additionalProperties.description": "Value for the environment variable",
+    "cmake-tools.configuration.cmake.configureEnvironment.description": "Environment variables to pass to CMake during configure",
+    "cmake-tools.configuration.cmake.buildEnvironment.description": "Environment variables to pass to CMake during build",
+    "cmake-tools.configuration.cmake.testEnvironment.description": "Environment variables to pass to CTest",
+    "cmake-tools.configuration.cmake.mingwSearchDirs.description": "Directories where MinGW may be installed",
+    "cmake-tools.configuration.cmake.searchDirs.items.description": "Path to a directory",
```

Comment:
I don't really know what this one is based on the description 😄.

Reply:
It's within a list of items in the context of another configuration field, so seems like it would be sufficiently described, in context.